contributingGuides/NAVIGATION.md

```diff
@@ -536,7 +536,7 @@ Here are examples how the state is generated based on route:
                         "params": {
                             "policyID": "1"
                         },
-                        "path": "/settings/workspaces/1/overview",
+                        "path": "workspaces/1/overview",
```

Comment:
We should also add this new path to [apple-app-site-association
](https://github.com/Expensify/App/blob/15aa1ad2f78bd9fa3bd0f17218f43e2ce06a52f3/.well-known/apple-app-site-association#L7-L18) file, so our smart app banner can display on mSafari.

More details:
https://github.com/Expensify/App/issues/66743#issuecomment-3105550005